tree.go

```diff
@@ -173,6 +183,35 @@ func addFdsToTree(tree treeprint.Tree, fds []*crit.Fd) {
 	}
 }
 
+func addSkToTree(tree treeprint.Tree, sks []*crit.Sk) {
+	var node treeprint.Tree
+	for _, sk := range sks {
+		node = tree.FindByMeta(sk.PId)
+		// If FindByMeta returns nil, then the node with
+		// the PID has been pruned while building the tree.
+		// Hence, skip all associated sockets.
+		if node == nil {
+			continue
+		}
+
+		var data string
+		for _, socket := range sk.Sockets {
+			switch socket.FdType {
+			case "INETSK":
+				data = fmt.Sprintf("%s:%d -> %s:%d [%s]", socket.SrcAddr, socket.SrcPort, socket.DestAddr, socket.DestPort, socket.State)
+			case "UNIXSK":
+				data = fmt.Sprintf("%s [%s]", socket.SrcAddr, socket.State)
+			case "PACKETSK":
+				data = fmt.Sprintf("↑ %s ↓ %s", socket.SendBuf, socket.RecvBuf)
+			case "NETLINKSK":
+				data = fmt.Sprintf("↑ %s ↓ %s [%s]", socket.SendBuf, socket.RecvBuf, socket.State)
+			}
+
+			node.AddMetaBranch(fmt.Sprintf("%s (%s)", socket.Type, socket.Protocol), data)
```

Comment:
`socket.Protocol` is not set for Unix sockets.
```
jira
├── Image: docker.io/atlassian/jira-software:latest
├── ID: 3f731c3a3951e25cc6509a3d49d7e8b799dc14dec4a96abb95e1ece55fb1692c
├── Runtime: crun
├── Created: 2023-08-11T10:24:48+01:00
├── Engine: Podman
├── Checkpoint size: 495.1 MiB
├── Root FS diff size: 74.5 KiB
└── Process tree
    └── [1]  tini
        └── [2]  java
            ├── [STREAM ()]   [ESTABLISHED]
            ├── [STREAM (TCP)]  :::8080 -> :::0 [LISTEN]
            ├── [STREAM ()]   [ESTABLISHED]
            └── [STREAM (TCP)]  7f00:1:::8005 -> :::0 [LISTEN]
```

Reply:
Oops my bad, will get rid of that